webapp/src/components/post_type/component.tsx

```diff
@@ -48,60 +48,65 @@ const PostType = ({post, connectedID, hasCall, pictures, profiles, showSwitchCal
         <Duration>{moment(post.props.start_at).fromNow()}</Duration>
     );
 
+    const title = post.props.title ? (<h3 className='markdown__heading'>{post.props.title}</h3>): (<div></div>);
```

Comment:
Let me know if there's a better to way signify an empty variable other than `<div></div>`. :smiling_face_with_tear:

Reply:
I think we can use [inline conditional rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) and only add the case we are interested in. So no need to initialize a variable but simply use this on top of the render return body:

```tsx
            {post.props.title &&
                <h3 className='markdown__heading'>{post.props.title}</h3>
            }
```

Reply:
Nice. Learnt so much!